### PR TITLE
Options/inizialization

### PR DESCRIPTION
**Pull request by Artefall**
Project was inizializated with Neutrino and using preset '@neutrinojs/node'.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-plugin-naming-convention",
+  "version": "1.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "neutrino start --use @neutrinojs/node",
+    "build": "neutrino build --use @neutrinojs/node"
```
Comment (constgen) [inline package.json]: We don't need `--use` flag when we have `.neutrinorc` file in the project root. All information is taken from the file autimatically

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "git+https://github.com/atomspace/eslint-plugin-naming-convention.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atomspace/eslint-plugin-naming-convention/issues"
+  },
+  "homepage": "https://github.com/atomspace/eslint-plugin-naming-convention#readme",
+  "devDependencies": {
+    "@neutrinojs/node": "^8.3.0",
```
Comment (constgen) [inline package.json]: Let's better use fixated versions for all packages to avoid unnecessary braking changes

**Comment by constgen** [comment]
We need also `@atomspace/eslint` and `@neutrinojs/jest` in this project

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "eslint-plugin-naming-convention",
+  "version": "1.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (constgen) [inline package.json]: this should call `neutrino test` command

Comment (Artefall) [reply package.json]: Neutrino test command was added

**Pull request merged**